docs/src/manuals/inference/undefinedrules.md

```diff
@@ -0,0 +1,245 @@
+# [Inference without explicit message update rules](@id inference-undefinedrules)
+
+`RxInfer` utilizes the [`ReactiveMP.jl`](https://github.com/ReactiveBayes/ReactiveMP.jl) package as its inference backend. Typically, running inference with `ReactiveMP.jl` requires users to define a factor node using the `@node` macro and specify corresponding message update rules with the `@rule` macro. Detailed instructions on this can be found in [this section](@ref create-node) of the documentation. However, in this tutorial, we will explore an alternative approach that allows inference with default message update rule for custom factor nodes by defining only `BayesBase.logpdf` and `BayesBase.insupport` for a factor node, without needing explicit `@rule` specifications.
+
+!!! note 
+    In the context of message-passing based Bayesian inference, custom message update rules enhance precision and efficiency. These rules leverage the specific mathematical properties of the model's distributions and relationships, leading to more accurate updates and faster convergence. By incorporating domain-specific knowledge, custom rules improve the robustness and reliability of the inference process, particularly in complex models where default rules may be inadequate or inefficient.
+
+## A simple prior-likelihood model
+
+We start a simple model with a hidden variable `p` and observations `y`. Later in the tutorial we explore more advanced use-cases.
+In this particular case we assume that `p` follows a prior distribution and `y` are drawn from a likelihood distribution. The model can be defined as follows:
+
+```@example inference-undefinedrules
+using RxInfer
+
+@model function simple_model(y, prior, likelihood)
+    p ~ prior
+    y .~ likelihood(p)
+end
+```
+
+## Node specifications
+
+Next, we define structures for both the `prior` and the `likelihood`. Let's start with the prior. Assume that the `p` parameter is best described by a `Beta` distribution. We can define it as follows:
+
+!!! note
```

Comment:
I would just make it slightly clearer.

**Note**: The `Distributions.jl` package already provides a fully-featured implementation of Beta distributions, including functions like `logpdf` and support checks. The example below redefines the Beta distribution structure and related functions solely for illustrative purposes. In practice, you do not need to define these functions yourself, as they are already included in `Distributions.jl`.